### PR TITLE
Add unit tests for CDCImporterManager

### LINKED ISSUE #28539: Add more unit tests for CDCImporterManager
Hi community,

This issue is for https://github.com/apache/shardingsphere/discussions/28487.

## Aim
Add more unit tests for CDCImporterManager to test its public functions to improve unit test coverage.

Unit test coverage can be viewed at: https://app.codecov.io/gh/apache/shardingsphere

## Basic Qualifications
- Java
- Maven
- Junit

### PR DESCRIPTION
**Pull request by isdrchagas**
Fixes #28539

Changes proposed in this pull request:
  - Add unit tests for CDCImporterManager

---

Before committing this PR, I'm sure that I have checked the following options:
- [x] My code follows the [code of conduct](https://shardingsphere.apache.org/community/en/involved/conduct/code/) of this project.
- [x] I have self-reviewed the commit code.
- [x] I have (or in comment I request) added corresponding labels for the pull request.
- [x] I have passed maven check locally : `./mvnw clean install -B -T1C -Dmaven.javadoc.skip -Dmaven.jacoco.skip -e`.
- [ ] I have made corresponding changes to the documentation.
- [ ] I have added corresponding unit tests for my changes.

### CODE REVIEW

File: kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    }
+
+    @Test
+    void assertRemoveImporter() {
+        CDCImporterManager.putImporter(actual);
+        CDCImporterManager.removeImporter(actual.getImporterId());
+        assertNull(getImporter(actual.getImporterId()));
+    }
+
+}
```
Comment (azexcy) [inline kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java:61]: Could we add new line at end of file? It's a checkstyle rule

File: kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.cdc.core.importer;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.apache.shardingsphere.data.pipeline.cdc.core.importer.CDCImporterManager.getImporter;
```
Comment (azexcy) [inline kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java:23]: image

It also looks like it violates the checkstyle, 
> Only those that relate to Mockito, junit Assertions, hamcrest CoreMatchers and MatcherAssert can use static import.

Comment (isdrchagas) [reply kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java]: my mistake, sorry for not adhering to some of the checkstyle rules in my recents code submissions! I've now corrected it here https://github.com/apache/shardingsphere/pull/28666/commits/a8c659f7ff1112b0555506475d3fef4db36440c0

if there's something else, let me know :)

Comment (azexcy) [reply kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java]: Ok, there is one more, I think this should be the last one :)
> https://github.com/apache/shardingsphere/actions/runs/6456714607/job/17526789037?pr=28666
Run 'mvn spotless:apply' to fix these violations.

Comment (isdrchagas) [reply kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java]: for some reason i'm not able to run the command `mvn spotless:apply`
image

is there something I'm missing here?

Comment (azexcy) [reply kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java]: image

`./mvnw spotless:apply -Pcheck` It needs to be executed in the root directory of the project

Comment (isdrchagas) [reply kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java]: It worked, thank you very much for the help!

Comment (azexcy) [reply kernel/data-pipeline/scenario/cdc/core/src/test/java/org/apache/shardingsphere/data/pipeline/cdc/core/importer/CDCImporterManagerTest.java]: you're welcome, thank you for your contribution.

**Review by azexcy — APPROVED** [review]

**Pull request merged**